install/kubernetes/Makefile

```diff
@@ -7,7 +7,7 @@ MANAGED_ETCD_VERSION := "v2.0.7"
 
 QUICK_INSTALL := "$(ROOT_DIR)/$(RELATIVE_DIR)/quick-install.yaml"
 EXPERIMENTAL_INSTALL := "$(ROOT_DIR)/$(RELATIVE_DIR)/experimental-install.yaml"
-MANAGED_ETCD_PATH := "$(ROOT_DIR)/$(RELATIVE_DIR)/cilium/charts/managed-etcd/values.yaml"
+MANAGED_ETCD_PATH := "$(CILIUM_CHARTS)/values.yaml"
 CILIUM_CHARTS := "$(ROOT_DIR)/$(RELATIVE_DIR)/cilium"
```

Comment:
Nit: not sure if this will cause problems but `CILIUM_CHARTS` is defined after it's used.

Reply:
I actually think this value can be removed. It is only consumed in a single step later to update the appVersion for the etcd sub-chart, but that component is removed and now part of cilium.